src/components/Display.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function Display(state) {
+  return (
+    <div>
+      <h1>{state.result}</h1>
+    </div>
+  );
+}
```

Comment:
please validate that result is a string ![screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)

Also use the defaultprop value in react to assign a defualt value to the result prop 
![screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)